model/ConceptProperty.php

```diff
@@ -47,7 +50,7 @@ public function getLabel()
         }
 
         // if not, see if there was a label for the property in the graph
-        if ($this->label) {
+        if ($this->label !== null) {
```

Comment:
Just a side note: I can see why this was done, but does not - nevertheless - change the code behavior a bit. But one can argument for code quality measures, and I agree.

Reply:
I copied and pasted some of the code for the `label` method into `description` and noticed this potential problem, so I fixed it in both places. In PHP, a string containing a zero `"0"` evaluates as false. It's an unlikely label, I know, but still, the behavior does change a little bit here ;)